app/components/DesignComponent/index.js

```diff
@@ -153,12 +156,12 @@ export default class Design extends Component<Props, State> {
     if (this.state.isPreviewing) {
```

Comment:
Change this so that the preview shows up in the same screen as the other content. See Kim's designs